atlasaction/action.go

```diff
@@ -747,6 +824,82 @@ func (a *Actions) SchemaApply(ctx context.Context) error {
 	return nil
 }
 
+// MonitorSchema runs the Action for "ariga/atlas-action/monitor/schema"
+func (a *Actions) MonitorSchema(ctx context.Context) error {
+	db, err := url.Parse(a.GetInput("url"))
+	if err != nil {
+		return err
+	}
```

Comment:
Maybe add `GetURLInput` to parse and validate the URL. Beware, invalid URL will be print the full string in its error.

Reply:
I mean add new helper for handle URL input and validate it. :D